tokio/src/sync/mpsc/error.rs

```diff
@@ -36,6 +36,16 @@ pub enum TrySendError<T> {
     Closed(T),
 }
 
+impl<T> TrySendError<T> {
+    /// Consume the `TrySendError`, returning the unsent value
```

Comment:
Applies both places.
```suggestion
    /// Consume the `TrySendError`, returning the unsent value.
```